aten/src/ATen/native/native_functions.yaml

```diff
@@ -5285,6 +5285,9 @@
     CUDA: cat_out_cuda
     QuantizedCPU: quantized_cat_out
 
+- func: _foreach_add.Scalar(Tensor[] tensors, Scalar scalar) -> Tensor[]
```

Comment:
I personally find "foreach" verbose. The add operation is already well defined and the only difference here is that you pass a Tensor[] instead of a Tensor. I'd opt for simply "_add".

Reply:
The intent for "foreach" is to amplify that we are working with Tensor Lists. But im open to the idea of overloading the operators down the road.

Reply:
this is discussed in the original issue: https://github.com/pytorch/pytorch/issues/38655.